.github/actions/setup-builder/action.yaml

```
@@ -51,7 +55,11 @@ runs:
       shell: bash
       run: |
         echo "Installing ${{ inputs.rust-version }}"
-        rustup toolchain install ${{ inputs.rust-version }}
+        rustup toolchain install ${{ inputs.rust-version }} --target ${{ inputs.target }}
         rustup default ${{ inputs.rust-version }}
-        rustup target add wasm32-unknown-unknown
-        echo "CARGO_TARGET_DIR=/github/home/target" >> $GITHUB_ENV
+    - name: Disable debuginfo generation
```

```suggestion
    - name: Disable debuginfo generation
    # Disable full debug symbol generation to speed up CI build and keep memory down
    # "1" means line tables only, which is useful for panic tracebacks.    
```

I think the rationale is important so it doesn't get lost accidentally along the way